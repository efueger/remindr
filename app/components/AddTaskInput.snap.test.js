import React from 'react';
import renderer from 'react-test-renderer';

import AddTaskInput from './AddTaskInput';

describe('Add Task Input', () => {
  it('renders the component', () => {
    const component = renderer.create(
      <AddTaskInput />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
