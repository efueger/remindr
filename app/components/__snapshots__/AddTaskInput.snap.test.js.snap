exports[`Add Task Input renders the component 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderTopColor": "#d9dbdb",
      "borderTopWidth": 1,
      "padding": 10
    }
  }>
  <TextInput
    autoCapitalize="sentences"
    blurOnSubmit={true}
    enablesReturnKeyAutomatically={true}
    onChange={[Function bound onChange]}
    onSubmitEditing={[Function bound onSubmit]}
    style={
      Object {
        "height": 19
      }
    }
    value="" />
</View>
`;
